From 5fd12b84dad423b0482f663a844b44ab1f6fc5f6 Mon Sep 17 00:00:00 2001
From: Feng Xiao <xfxyjwf@gmail.com>
Date: Thu, 11 Dec 2014 15:17:04 -0800
Subject: [PATCH] Update gtest version to v1.7.0.

---
 autogen.sh | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/autogen.sh b/autogen.sh
index 8e455e9..c18474b 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -18,9 +18,11 @@ fi
 # Check that gtest is present.  Usually it is already there since the
 # directory is set up as an SVN external.
 if test ! -e gtest; then
-  echo "Google Test not present.  Fetching gtest-1.5.0 from the web..."
-  curl http://googletest.googlecode.com/files/gtest-1.5.0.tar.bz2 | tar jx
-  mv gtest-1.5.0 gtest
+  echo "Google Test not present.  Fetching gtest-1.7.0 from the web..."
+  curl -O https://googletest.googlecode.com/files/gtest-1.7.0.zip
+  unzip -q gtest-1.7.0.zip
+  rm gtest-1.7.0.zip
+  mv gtest-1.7.0 gtest
 fi
 
 set -ex
